net/tstun/wrap.go

```diff
@@ -64,7 +65,7 @@ type Wrapper struct {
 
 	closeOnce sync.Once
 
-	lastActivityAtomic int64 // unix seconds of last send or receive
+	lastActivity mono.Time // time of last send or receive, accessed atomically
```

Comment:
keep the old variable name? the Atomic suffix is ugly and scary which is a feature.

I guess you didn't like the stutter in `lastActivityAtomic.LoadAtomic` etc? I'm fine with it if you are.

Reply:
Yeah, I was removing the stutter, but don't feel strongly. Re-added.